app/src/main/res/values/strings.xml

```diff
@@ -269,10 +264,10 @@
     <!-- Feature discovery -->
     <string name="app_info_title">How The Algorithm Works</string>
     <string name="app_info_body">
-        1. You choose between two users repeatedly.\n
-        2. The app then creates a stack ranking.\n
-        3. A daily algorithm matches for mutual attraction.\n
-        4. Enjoy your top 10 matches!
+        1. Choose between two profiles repeatedly.\n
+        2. App creates profile rankings.\n
```

Comment:
I wonder if there is a way to tie #2 to #1, i.e. to suggest that the profile comparisons result in the ranking rather than the app deciding on the ranking.

"Stack ranking of profiles is inferred." (clumsy wording)